### PR TITLE
Save cognito_id_token in cookie

### PR DESCRIPTION
**Pull request by Ulthran**
Not sure if these changes are within the scope of the package as you see it, but I had to spend a while learning about Cognito and the many ways of authenticating to AWS services so figured I'd include this to make it easier on the next.

FYI, my use case is an AWS Amplify app that I then want to have a separate flask site with access to all the components. Not sure if that should be mentioned in the docs or not.

### CODE REVIEW

**Comment by mblackgeo** [comment]
Hey apologies for the delay in getting round to this - are you able to separate out the functional changes from documentation into separate PRs please?

**Comment by planestoner** [comment]
Hi, is there any progress on this PR as it would be very beneficial and appears similar to that which was reported in issue #37? If I'd be willing to submit the PRs but would you be able to clarify the request 
> separate out the functional changes from documentation into separate PRs  please?

Are you requesting the a PR for the changes to [config.py](https://github.com/mblackgeo/flask-cognito-lib/pull/45/commits/bf205d80a4680ad132c7a8467935a5d78ae405ad#diff-9f77a21c6dd44ab8f804ddea2246e5d20cdde11f6cc9ec88e9f971a02d60552f) and [decorators.py](https://github.com/mblackgeo/flask-cognito-lib/pull/45/commits/bf205d80a4680ad132c7a8467935a5d78ae405ad#diff-9f77a21c6dd44ab8f804ddea2246e5d20cdde11f6cc9ec88e9f971a02d60552f) in one PR and then the [docs/accessing-aws-resources.md](https://github.com/mblackgeo/flask-cognito-lib/pull/45/commits/bf205d80a4680ad132c7a8467935a5d78ae405ad#diff-27030f38f732908b722565af341aff8fa7a7de60296d1992b70295d3c08dae0f) in a separate PR?

**Comment by Ulthran** [comment]
Done with #50!

File: src/flask_cognito_lib/decorators.py [hunk 1]
```diff
@@ -252,6 +270,13 @@ def wrapper(*args, **kwargs):
                     domain=cognito_auth.cfg.cookie_domain,
                 )
 
+            # Remove the id token if it exists
+            if get_token_from_cookie(cognito_auth.cfg.COOKIE_NAME_ID):
+                resp.delete_cookie(
+                    key=cognito_auth.cfg.COOKIE_NAME_ID,
```
Comment (mblackgeo) [inline src/flask_cognito_lib/decorators.py:276]: Would you be able to add some test coverage of these new conditional statements please?

**Review by mblackgeo — COMMENTED** [review]
Thanks for the PR!

Please would you be able to add some test coverage, thanks!

File: src/flask_cognito_lib/decorators.py (hunk 1 above)
Comment (Ulthran) [reply src/flask_cognito_lib/decorators.py:276]: See commit below

**Comment by mblackgeo** [comment]
Thanks for the contribution - now release in v1.9.0 :tada:

**Pull request merged**